index.html

```diff
@@ -1688,40 +1698,62 @@
             </dd>
           </dl>
           <ol>
-            <li>If the <a>presentation connection state</a> of
-            <code>presentationConnection</code> is not <code>connected</code>
-            or <code>connecting</code>, then abort these steps.
+            <li>Signal the intention to close the corresponding
+            <code>PresentationConnection</code> to the <a>destination browsing
+            context</a>, passing the <code>closeReason</code> to that context.
             </li>
```

Comment:
I'll try to clarify that the event loop should not block on receiving an ack from the destination user agent.  

Yes the steps in "start closing the presentation connection" should run in the same event loop when close() is called.   This seems to be implicit in this spec, i.e. it's assumed to be the case if we don't specify that we are queuing a task to create a new event loop.